business_requirement_deliverable/models/business.py

```diff
@@ -194,9 +194,13 @@ def _compute_dl_total_revenue(self):
         for r in self:
             r.dl_total_revenue = sum(r.deliverable_lines.mapped("price_total"))
 
+    @api.depends_context("website_id")
     def _compute_dl_count(self):
         for r in self:
-            r.dl_count = len(r.deliverable_lines.ids)
+            if "website_id" in self.env.context:
```

Comment:
I don't see why you need the website here, as this doesn't depend on website nor the field is depending on it.